src/components/Loader/__tests__/loader-snapshots.jest.module.scss

```diff
@@ -0,0 +1,4 @@
+.customLoaderSize {
+  height: 100px;
+  width: 100px;
```

Comment:
I know its not related to your pr but can you fix the linter error in editable heading? Because it fails your PR.